apps/ui/src/components/ButtonFollow.vue

```diff
@@ -1,5 +1,9 @@
 <script setup lang="ts">
 import { Space } from '@/types';
+import { METADATA } from '@/networks/starknet';
+
+// NOTE: Disabling on Starknet until we have support for starknet long addresses on sequencer
+const DISABLED_NETWORKS = Object.keys(METADATA);
```

Comment:
It's a bit hacky to get list of starknet networks through this, can we instead make it a whitelist and use already defined constants `evmNetworks` and `offchainNetworks` as `SUPPORTED_NETWORKS`?